.github/workflows/api.yml

```diff
@@ -12,6 +12,7 @@ env:
   LOG_CHANNEL: stack
   LOG_LEVEL: debug
   DB_CONNECTION: sqlite
+  JWT_SECRET: veCMWIG3Mv6c1w5pzBs4FRp4fBAiYcV5e6rDeiHVW9iZeO4cuYOJeT8CXJ2caQ3f
```

Comment:
Shouldn't the secret key be hidden and loaded dynamically e.g. from environment variables?

Reply:
Not here, it is not secret key for external provider that needs to be protected. It is simply used as part of hashing process, so as long as you set it to other value in staging or production env, this key is useless for attacker.